### PR TITLE
Increase the severity of no-console rule inside packages folder.

### PR DESCRIPTION
**Pull request by bipuladh**
- Increases severity of `no-console` to 2.

### CODE REVIEW

File: frontend/packages/.eslintrc.js [hunk 1]
```diff
@@ -9,5 +9,6 @@ module.exports = {
         message: 'Use lodash instead. webpack is configured to use lodash-es automatically.',
       },
     ],
+    'no-console': 2,
```
Comment (benjaminapetersen) [inline frontend/packages/.eslintrc.js]: nit: I'd prefer "error" to `2` personally.

Comment (christianvogt) [reply frontend/packages/.eslintrc.js]: Please move `no-console: 'error'` to `airbnb-base-overrides.js`

**Pull request merged**